ajax/libs/cellx/package.json

```
@@ -1,7 +1,7 @@
 {
   "name": "cellx",
-  "filename": "cellx.min.js",
-  "version": "1.7.57",
+  "filename": "cellx.umd.js",
+  "version": "1.7.82",
```

```suggestion
  "version": "1.7.57",
```

Please revert the change to `version`, the auto-update bot will handle chaging this when it pulls in new versioned files.